homeassistant/components/pjlink/media_player.py

```diff
@@ -36,6 +38,7 @@
         vol.Optional(CONF_NAME): cv.string,
         vol.Optional(CONF_ENCODING, default=DEFAULT_ENCODING): cv.string,
         vol.Optional(CONF_PASSWORD): cv.string,
+        vol.Optional(CONF_TIMEOUT, default=DEFAULT_TIMEOUT): cv.positive_int,
```

Comment:
We don't want to add options for timeouts. We should set sane defaults.

What is the problem at hand?

Reply:
The underlying library used to handle the PJLink protocol was recently updated with a default timeout of 2 seconds but it seems to be too short for some projectors.

I can remove the timeout config and set a default value instead.